script/DeployMaxVault.s.sol

```diff
@@ -139,6 +139,7 @@ contract DeployMaxVault is BaseScript {
 
         // wbnb
         setWethDepositRule(vault, contracts.WBNB());
+        setWithdrawAssetRule(vault, contracts.YNCLISBNBK(), contracts.WBNB());
```

Comment:
We need to write a weth withdraw rule, what you have added is a withdrawAsset rule